tide/master/localstreamer/CommandLineOptions.cpp

```diff
@@ -203,7 +193,7 @@ void CommandLineOptions::_parseCommandLineArguments( int& argc, char** argv )
     }
 
     _getHelp = vm.count( "help" );
-    _streamname = vm["streamname"].as<std::string>().c_str();
+    _streamId = vm["streamname"].as<std::string>().c_str();
```

Comment:
change the commandline option name as well? Or why the 'inconsistency' is introduced?